libagent/ssh/__init__.py

```diff
@@ -114,6 +114,10 @@ def create_agent_parser(device_type):
 
     p.add_argument('identity', type=_to_unicode, default=None,
```

Comment:
I think it would be simpler to add `nargs='+'` to the existing `identity` argument:
```python
p.add_argument('identity', type=_to_unicode, default=None, nargs='+',
               help='proto://[user@]host[:port][/path]')
```
See https://docs.python.org/3/library/argparse.html#nargs

Reply:
Let me test how that works.